cruise-control-core/src/main/java/com/linkedin/cruisecontrol/monitor/sampling/aggregator/MetricSampleAggregator.java

```diff
@@ -567,4 +574,18 @@ private long windowIndex(long time) {
   private E identity(E entity) {
     return _identityEntityMap.computeIfAbsent(entity, e -> entity);
   }
+
+  /**
+   * Get the length of time aggregator keeps sample in memory.
+   * It is used to determine the time length to look back(from current time) when loading the historical samples.
```

Comment:
`It is used to determine the time length to look back(from current time) when loading the historical samples.` -> This seems to be one of many potential ways of using this method -- i.e. we can use it for other purposes in the future.
Would we avoid specifying our particular use-case in the JavaDoc of this generic method?